notebooks/quickstart_tutorial.ipynb

```diff
@@ -313,9 +313,17 @@
+    ">**Tooltip:** The visualization displays various cell type ontology terms as colored circles in a circular dendrogram. The relationships underlying this dendrogram correspond to \"_is_a_\" relationships from [Cell Ontology](https://obofoundry.org/ontology/cl.html) (CL). Since these relationships are not mutually exclusive, a term can have multiple parent terms, meaning the same term can appear along different branches of the tree representation. The radius of each circle (whether it is a clade or a leaf node) signifies the occurrence of the term in the entire dataset, regardless of its relevance score. The color of the circle indicates the relevance score of the term in cells where it was found to have non-vanishing relevance.\n",
+    ">\n",
+    "> Here are some of the interactive capabilities of the visualization app:\n",
+    "> - **Cell selection:** By default, all cells are selected, and the cell type ontology dendrogram shows an aggregated summary over all cells. You can restrict the aggregation to a subset of cells by selecting your desired subset over the UMAP scatter plot using the rectangular select or lasso select tool. The dendrogram will react to your custom cell selection. If your input AnnData file includes clustering, you can restrict score aggregation to each cluster by selecting your cluster in the Settings panel (accessible via the gear icon in the upper right of the app). \n",
+    "> - **Highlighting ontology term relevance scores:** You can highlight cell type ontology term relevance scores over the UMAP scatter plot by clicking on the circles in the dendrogram. Only the selected cells will be scored, and the rest will be grayed out. You can revert to selecting all cells from the settings panel or by using the rectangular select tool to select all cells.\n",
+    "> - **Studying the ontology term relevance scores for a single cell:** You can display the term relevance scores for individual cells by clicking on a single cell in the UMAP scatter plot.\n",
+    "> - **Advanced settings:** By default, only terms above a specified relevance threshold with occurence above another threshold over the selected cells are shown. You can modify these threshold in the Settings panel (accessible via the gear icon in the upper right of the app).\n",
+    ">\n",
+    ">**Note**: The number of cells displayed should be limited to roughly 50K. Beyond that performance may suffer.  If you need to visualize more cells, please attempt to downsample your cells."
```

Comment:
```suggestion
    ">**Note**: The number of cells displayed should be limited to roughly 50K. Beyond that, performance may suffer.  If you need to visualize more cells, please attempt to downsample your cells."
```

The biggest possible nitpick.   I'll know I've finally gone too far when I start policing Oxford commas.

Reply:
haha yes!  thanks man!